teacher.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'person'
+
+# Class representing a student
+class Teacher < Person
+  def initialize(name, age, specialization)
+    super(name, age)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    trues
```

Comment:
please make sure to correct this one from `trues to true ` in order to return true

Reply:
Thank you for your review @youmari. Done!